echemdb/data/cv/entry.py

```diff
@@ -175,29 +175,29 @@ def format_title(self, e, which_field, as_sentence=True):
 
     def x(self):
         r"""
-        Return the name of the variable on the x-axis, i.e., `"U"`.
-
-        TODO: Adapt along with https://github.com/echemdb/svgdigitizer/issues/106.
+        Return the name of the variable on the x-axis, i.e., `"E"`.
 
         EXAMPLES::
 
             >>> entry = Entry.create_examples()[0]
             >>> entry.x()
-            'U'
+            'E'
 
         """
         from astropy import units as u
-        if u.Unit(self.figure_description.potential_scale.unit).is_equivalent('V'):
-            return 'U'
+
+        if not self.data_description.axes:
+            raise ValueError(f"No axes were specified for the dataset.")
```

Comment:
I think this is not the appropriate exception here since there is no argument. From the Python docs:
```
Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.
```